test/default-xpring-client-test.ts

```diff
@@ -401,4 +401,34 @@ describe('Default Xpring Client', function(): void {
       done()
     })
   })
+
+  it('Check if account exists - exists with valid address and positive balance', async function() {
```

Comment:
It would be cool to add an integration test for this too! See `test/integration-tests.ts`.

(You can do this in a separate PR if you'd like)

Reply:
will do in separate PR